src/seedu/addressbook/logic/Logic.java

```diff
@@ -85,7 +85,10 @@ public CommandResult execute(String userCommandText) throws Exception {
     private CommandResult execute(Command command) throws Exception {
```

Comment:
You need to update the header comment for Logic#execute() method as the behavior of execute changes due to the way data is saved.